src/index.ts

```diff
@@ -1,3 +1,4 @@
+import { Buffer } from "buffer"
```

Comment:
And that's the fix?  TS and JS really is weird. xD

Reply:
no, the fix is the export not the import - but it needs to be imported to be exported!